gt/main.c

```diff
@@ -45,10 +45,6 @@
 #include "gatekeeper_l2.h"
 #include "gatekeeper_varip.h"
 
-/* TODO Get the install-path via Makefile. */
-#define LUA_POLICY_BASE_DIR "./lua"
-#define GRANTOR_CONFIG_FILE "policy.lua"
```

Comment:
These original macro names were a little bit confusing, and I think we should use this patch as an opportunity to change them. `LUA_POLICY_BASE_DIR` maybe should become `gt_conf->lua_base_directory` and `GRANTOR_CONFIG_FILE` should maybe become `gt_conf->lua_policy_file`.